exchanges/binance/cfutures_types.go

```diff
@@ -388,6 +388,8 @@ type FuturesAccountInformation struct {
 		PositionSide           string  `json:"positionSide"`
 		EntryPrice             float64 `json:"entryPrice,string"`
 		MaxQty                 float64 `json:"maxQty,string"`
+		PositionAmount         float64 `json:"positionAmt,string"`
+		UpdateTime             int64   `json:"updateTime"`
```

Comment:
This is a few steps, but we've got support for these binance times, so we may as well keep using it. Check out `binance/type_convert.go`

So my recommendation here is to split out the `positions` json into its own struct and use it instead of the inline struct in `FuturesAccountInformation`:
```
// FuturesAccountInformationPositions  holds account position data
type FuturesAccountInformationPositions struct {
	Symbol                 string    `json:"symbol"`
	InitialMargin          float64   `json:"initialMargin,string"`
	MaintMargin            float64   `json:"maintMargin,string"`
	UnrealizedProfit       float64   `json:"unrealizedProfit,string"`
	PositionInitialMargin  float64   `json:"positionInitialMargin,string"`
	OpenOrderInitialMargin float64   `json:"openOrderInitialMargin,string"`
	Leverage               float64   `json:"leverage,string"`
	Isolated               bool      `json:"isolated"`
	PositionSide           string    `json:"positionSide"`
	EntryPrice             float64   `json:"entryPrice,string"`
	MaxQty                 float64   `json:"maxQty,string"`
	PositionAmount         float64   `json:"positionAmt,string"`
	UpdateTime             time.Time `json:"updateTime"`
}
```

Then in `type_convert.go` add the following:
```
// UnmarshalJSON deserialises the JSON info, including the timestamp
func (a *FuturesAccountInformationPositions) UnmarshalJSON(data []byte) error {
	type Alias FuturesAccountInformationPositions
	aux := &struct {
		UpdateTime binanceTime `json:"updateTime"`
		*Alias
	}{
		Alias: (*Alias)(a),
	}
	if err := json.Unmarshal(data, &aux); err != nil {
		return err
	}
	a.UpdateTime = aux.UpdateTime.Time()
	return nil
}
```

And now the update time is a proper `time.Time` which you can test using Binance's example and you should get a time of ~`2021-07-23 17:54:41.327 +1000 AEST`

An optional extra is to do the same with `FuturesAccountInformation` and you can get that `updateTime` as well